src/CMakeLists.txt

```diff
@@ -65,6 +65,12 @@ add_library(libJSBSim ${HEADERS} ${SOURCES}
   $<TARGET_OBJECTS:Simgear>
   $<TARGET_OBJECTS:GeographicLib>
   )
+target_include_directories(
+  libJSBSim
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
Not sure why you would want to put `CMAKE_CURRENT_BINARY_DIR` and `CMAKE_INSTALL_INCLUDEDIR` in the include directories ?

All the required headers for using libJSBSim are in `CMAKE_CURRENT_SOURCE_DIR`.
And `CMAKE_INSTALL_INCLUDEDIR` reports an empty string on my side.

Reply:
target_include_directories(
  libJSBSim
  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
)

Reply:
> Ok, no problem. So... This is better:
> 
> ```cmake
> target_include_directories(
>   libJSBSim
>   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
> )
> ```

Yep ! Looks good to me 👍

Reply:
Done